rollup/circuitcapacitychecker/libzkp/Cargo.lock

```diff
@@ -23,7 +23,7 @@ dependencies = [
 [[package]]
 name = "aggregator"
 version = "0.1.0"
-source = "git+https://github.com/scroll-tech/zkevm-circuits.git?tag=v0.7.6#ea904e8da403a35153a310cee6435a892cc9cfc8"
+source = "git+https://github.com/scroll-tech/zkevm-circuits.git?tag=v0.8.0#a88264d2c6b1d4608a40ed34696e9f8364e94d10"
```

Comment:
I guess we won't merge this until we have `v0.8.1`, right?

Reply:
Yes. Will update to `v0.8.1`.